### DISCUSSION TITLE
the program not working on windows

### DISCUSSION THREAD

**Discussion by smrh77** [HemantKArya/BloomeeTunes#100] [Q&A]
Unfortunately, this program does not run on my laptop , meaning when I open the program and select a category, a black screen opens and remains in the loading state, and nothing is displayed.

**Comment by HemantKArya** [comment]
I guess this is some kind of bug so you can you open a issue thread with some more details like screen shots, environment and also if there is any error visible that also.

**Comment by smrh77** [comment]
Thank you for your response. As is evident in the recorded clip, the
program does not work on the system. Additionally, I tried it on two
laptops, and it did not work on either of them.
program troubleshooting
software compatibility issues
more technical details



program troubleshooting
software compatibility issues
more technical details
